app/globals.css

```diff
@@ -51,7 +51,7 @@
 
 body {
   color: var(--foreground);
-  background: var(--background);
+  background: url("../public/noise.png") lightgray 0% 0% / 148.5629677772522px 148.5629677772522px repeat;;
```

Comment:
this image is not loading.

Reply:
can you achieve this effect without using an image and using only css?

Reply:
It was quite tricky to it with pure css and most solutions suggested the used of a SVG, the repeated png is doing the job for now.